packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx

```diff
@@ -106,6 +106,69 @@ describe('VitalsHeader: ', () => {
     expect(mockLaunchWorkspace).toHaveBeenCalledTimes(1);
     expect(mockLaunchWorkspace).toHaveBeenCalledWith(patientVitalsBiometricsFormWorkspace);
   });
+
+  it('renders a normal header when all the values fall within normal range', async () => {
+    mockFhirVitalsResponse.entry[3].resource.valueQuantity.value = 79;
+    mockFhirVitalsResponse.entry[4].resource.valueQuantity.value = 119;
+    mockFhirVitalsResponse.entry[0].resource.valueQuantity.value = 69;
+    mockFhirVitalsResponse.entry[11].resource.valueQuantity.value = 36;
+    mockOpenmrsFetch.mockReturnValue({ data: mockFhirVitalsResponse });
+
+    renderVitalsHeader();
+
+    await waitForLoadingToFinish();
+
+    const expandButton = screen.getByTitle(/ChevronDown/);
+    userEvent.click(expandButton);
+
+    expect(screen.queryByTitle(/abnormal value/i)).not.toBeInTheDocument();
+  });
+
+  it('renders a header with a pink backgroung when the diastolic value falls below the configured low normal range', async () => {
+    mockFhirVitalsResponse.entry[3].resource.valueQuantity.value = 69;
+
```

Comment:
Hi @nanfuka can you remove the extra line in 129

Reply:
A good pattern for writing unit tests is the [arrange-act-assert pattern](https://docs.telerik.com/devtools/justmock/basic-usage/arrange-act-assert). The use of a newline between the various sections of the test is a good practice for creating visual hierarchy and separation.